jobserver/templates/job_detail.html

```diff
@@ -158,16 +158,18 @@ <h3 class="text-center">Tools</h3>
         disabled
         {% endif %}
         type="submit">
-        Cancel (not working yet!)
+        Cancel
       </button>
     </form>
 
+    {% if request.user.is_superuser %}
     <form class="mb-3" method="POST" action="{{ job.get_zombify_url }}">
       {% csrf_token %}
 
       <button class="btn btn-block btn-danger" type="submit">Zombify</button>
     </form>
     {% endif %}
+    {% endif %}
```

Comment:
We'll want to end the authenticated and authorized user check at the end of the Cancel button, Zombification is only for us superusers still.

Reply:
I think the effect is that same either way (the superuser check is still there, just moved down) but your suggestion makes things clearer.

Reply:
Oh yes, I see what you mean now 👍